compiler/src/dotty/tools/dotc/config/Config.scala

```diff
@@ -160,7 +160,7 @@ object Config {
   final val showCompletions = false
 
   /** If set, enables tracing */
-  final val tracingEnabled = false
+  final val tracingEnabled = true
```

Comment:
Was this enabled on purpose?

Reply:
No, I mistakenly added two files to the commit that should not be there.